### PR TITLE
Food information

### PR DESCRIPTION
**Pull request by alexclaes**
I added some information about my favorite food

### CODE REVIEW

File: AboutMe.txt [hunk 1]
```diff
@@ -1,3 +1,5 @@
 Hi I'm Alex
 
 My favorite color is red
+
+My favorite food is pizza
```
Comment (doemser) [inline AboutMe.txt]: I would go for Spaghetti

**Review by doemser — CHANGES_REQUESTED** [review]
Please change food

File: AboutMe.txt (hunk 1 above)
Comment (alexclaes) [reply AboutMe.txt]: I changed the food

**Review by doemser — APPROVED** [review]
Looks good now.

**Pull request merged**